roles/sales-rep/map-view/js/customer.js

```diff
@@ -1,11 +1,12 @@
-define([], function () {
- 
-    function Select (bezl, custNum) {
+define(["./map.js"], function (map) {
+
+    function Select(bezl, custNum) {
+        console.log(bezl.vars.markers[custNum].data);
```

Comment:
@soft-Dev Do you want this console.log() call here or was this left over debugging?